extensions/wikia/SpecialUnsubscribe/Unsubscribe.body.php

```diff
@@ -41,42 +41,31 @@ public function execute( $subpage ) {
 
 		$hash_key = $wgRequest->getText('key', null );
 
-		$email = $token = $timestamp = null;
+		$email = $token = null;
 		if ( !empty( $hash_key ) ) {
 			#$hask_key = urldecode ( $hash_key );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)